src/libsyncengine/reconciliation/platform_inconsistency_checker/platforminconsistencycheckerworker.cpp

```diff
@@ -129,9 +129,6 @@ void PlatformInconsistencyCheckerWorker::blacklistNode(const std::shared_ptr<Nod
         case InconsistencyTypeNameLength:
             causeStr = L"the name is too long";
             break;
-        case InconsistencyTypePathLength:
```

Comment:
So, this is dead code?

Reply:
This inconsistency is never handled here. SonarCloud suggested that unused cases should be remove from the switch